compose.yaml

```diff
@@ -5,8 +5,6 @@ x-redash-service: &redash-service
     context: .
     args:
       skip_frontend_build: "true"  # set to empty string to build
-  volumes:
-    - .:/app
```

Comment:
This bindmount is not a mistake, it is required for live-reload in a dev environment

Reply:
@eradman what about using separate path for frontend artifacts like `/www/html` [here](https://github.com/getredash/redash/blob/0900178d245615ba52bcaf62c73203c7d99b896f/Dockerfile#L115) and [here](https://github.com/getredash/redash/blob/0900178d245615ba52bcaf62c73203c7d99b896f/redash/handlers/webpack.py#L6) to avoid shadowing by mounted volume? Let me know if you accept this approach

Reply:
We need to understand #7433 better. What is the reason for setting `skip_frontend_build: ""`?